src/wp-includes/css/admin-bar.css

```
@@ -97,6 +97,7 @@ html:lang(he-il) .rtl #wpadminbar * {
 	min-width: 600px; /* match the min-width of the body in wp-admin/css/common.css */
 	z-index: 99999;
 	background: #1d2327;
+	outline: 1px solid transparent;
```

```suggestion
	/* Only visible in Windows High Contrast mode */
	outline: 1px solid transparent;
```

As with any other style, it's a good idea to add comments.

---

Sure, Would add the comment to the change.